src/contracts/BaseCrossChainController.sol

```diff
@@ -61,11 +61,18 @@ contract BaseCrossChainController is
     );
   }
 
-  /// @inheritdoc IRescuable
-  function whoCanRescue() public view override(IRescuable, Rescuable) returns (address) {
+  /// @inheritdoc Rescuable
+  function whoCanRescue() public view override(Rescuable) returns (address) {
     return owner();
   }
 
+  /// @inheritdoc IRescuableBase
+  function maxRescue(
+    address erc20Token
+  ) public view override(IRescuableBase, RescuableBase) returns (uint256) {
+    return IERC20(erc20Token).balanceOf(address(this));
```

Comment:
why not just type(uint256).max?

Reply:
For what i understood of the readme this should return the max amount rescuable of the specified token